setup/tue-functions.bash

```diff
@@ -56,36 +53,23 @@ export -f _github_https_or_ssh # otherwise not available in sourced files
 
 function tue-make
 {
-    if [ -n "$TUE_ROS_DISTRO" ] && [ -d $_TUE_CATKIN_SYSTEM_DIR ]
+    if [ ! -d $TUE_SYSTEM_DIR ]
```

Comment:
In which script is `TUE_SYSTEM_DIR` being defined?

Reply:
See below; ros target